src/controllers/lucky.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from "express";
+import * as Services from "../dal/lucky";
+import { getText } from "../helpers/arrayProcessing";
+import { getTodayAllSignRecordsFromDB } from "../dal/zodiacdeep";
+
+export async function getLuckyTextByRandomNumber(
```

Comment:
The name of the function is **getLuckyTextByRandomNumber**, yet there is no random number passed. Something like **getLuckyZodiac** for name looks more appropriate;

Also this function can be put inside of **controllers/zodiac.ts** since it's related to zodiac